libs/angular/src/lib/accordion/accordion-list-item.component.html

```diff
@@ -0,0 +1,19 @@
+<ng-container data-testid="accordion-list-item-root">
+    <div role="heading" aria-level="2" [attr.id]="id">
+        <button data-testid="accordion-list-item-expander-button" [attr.id]="id + '_header'" (click)="toggleExpanded()"
+            tabindex="0" [attr.aria-expanded]="this.isExpanded" [attr.aria-controls]="id + '_section'">
+            <span data-testid="accordion-list-item-header">{{listItemHeader}}</span>
+            <span data-testid="accordion-list-item-subheader">{{listItemSubHeader}}</span>
+            <svg width="1em" height="1em" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path
+                    d="M18.8095 9.22817L18.1907 8.60942C18.0438 8.46255 17.8063 8.46255 17.6595 8.60942L12.0001 14.2563L6.34072 8.60942C6.19385 8.46255 5.95635 8.46255 5.80947 8.60942L5.19072 9.22817C5.04385 9.37505 5.04385 9.61255 5.19072 9.75942L11.7345 16.3032C11.8813 16.45 12.1188 16.45 12.2657 16.3032L18.8095 9.75942C18.9563 9.61255 18.9563 9.37505 18.8095 9.22817Z"
+                    fill="#333333" />
+            </svg>
+        </button>
+        <div role="region" class="accordion-list-item-content" [ngClass]="{'content-hidden' : !this.isExpanded}"
```

Comment:
We need to add the attribute hidden when closed otherwise users can tap into the content that is collapsed.

Reply:
Fixed